modal/sandbox.py

```diff
@@ -147,11 +147,33 @@ def __init__(self, sandbox_id: str, client: _Client):
         self._buffer = bytearray()
 
     def get_next_index(self):
+        """mdmd:hidden"""
         index = self._index
         self._index += 1
         return index
 
     def write(self, data: Union[bytes, bytearray, memoryview]):
+        """
+        This methods writes the data to an internal writer buffer waiting to be drained.
```

Comment:
```suggestion
        Write data to stream's internal buffer, but does not drain/flush the write.
```

minor rewording, but you also don't need to start a docstring with "This methods [sic]"